releases/v19.2.0-beta.20190910.md

```diff
@@ -113,6 +117,12 @@ $ docker pull cockroachdb/cockroach-unstable:v19.2.0-beta-20190903
 - Added `\d`, `\dt`, `\du`, `\l` metacommands to the [cockroach sql](../v19.2/use-the-built-in-sql-client.html) interactive shell. [#39141][#39141] {% comment %}doc{% endcomment %}
 - Added default cluster locality topologies for multi-node [`cockroach demo`](../v19.2/cockroach-demo.html) clusters. [#40221][#40221]
 - Errors in [`cockroach demo`](../v19.2/cockroach-demo.html) are no longer logged to the command-line interface. [#40196][#40196]
+- Added `\d` client-side command with no arguments as an alias for `\dt`, for more ease of adoption by `psql` users. [#40309][#40309]
+- Added a [MovR](../v19.2/movr.html) workload to run with [`cockroach demo`](../v19.2/cockroach-demo.html). [#40271][#40271]
+- Added location information for default localities to populate web UI for [`cockroach demo`](../v19.2/cockroach-demo.html). [#40229][#40229]
+- [`cockroach demo`](../v19.2/cockroach-demo.html) attempts to contact a license server to obtain a temporary license for demo usage. [#40273][#40273]
+- [`cockroach demo`](../v19.2/cockroach-demo.html) enables telemetry for demo clusters. You can opt out of telemetry by setting the `COCKROACH_SKIP_ENABLING_DIAGNOSTIC_REPORTING` environment variable (https://www.cockroachlabs.com/docs/stable/diagnostics-reporting.html). [#40273][#40273]
```

Comment:
`enables` > `now enables`

Also, what is the URL at the end for? Do you want to link to that page from some part of the note?